### ISSUE TITLE
Website mac download link

### ISSUE THREAD

**Issue by Lelelo1** [flutnet/flutnet-sdk#2]
**Website mac download**
The mac download link is down on the website: https://www.flutnet.com/Download/Flutnet-SDK.html

**Comment by masterwok** [comment]
@Lelelo1 were you ever able to resolve this? Just ran into this myself.

**Comment by Lelelo1** [comment]
@masterwok Nope. It's still down. I would get windows machine to develop with this - but then I hope this will maintained sufficiently.

**Comment by Lelelo1** [comment]
@masterwok 
Now website is down. Maybe this project is discontinued?
